ios/Ts_Assignment/Info.plist

```diff
@@ -4,6 +4,8 @@
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
+	<key>NSAppleMusicUsageDescription</key>
```

Comment:
why apple music?

Reply:
because it was giving an error and explicitly told me to add this

Reply:
Where it was asking, I mean which library?

Reply:
I was trying to install react-native-permissions. I cannot explain why! but it was not working without that! you can experience the same by trying to install react-native-permissions

Reply:
Please share the error that exactly you are facing.